<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/settingbg.png) 0 0 0 0 stretch stretch;</string>
  </property>
  <widget class="QCheckBox" name="isTraining">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;NiseSegaSonic&quot;;</string>
   </property>
   <property name="text">
    <string>Training Mode? 
(Should AI Fight Back?)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>GO!</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
